lib/exabgp/configuration/setup.py

```diff
@@ -230,6 +230,12 @@
 		},
 	},
 	'api':  {
+		'lines': {
```

Comment:
patatoes, tomatoes .... I do not have a really better idea, happy to go with what you have.